src/components/CardAboutShelter/components/InfoRow/InfoRow.tsx

```diff
@@ -37,6 +37,11 @@ const InfoRow = React.forwardRef<HTMLDivElement, IInfoRowProps>(
             {label}
           </span>
           <span className="md:flex">{ValueComp}</span>
+          {children ? (
+            children
+          ) : (
+            null
+          )}
```

Comment:
Aqui poderia ser simplesmente como abaixo não?
```suggestion
          {children}
```

Reply:
SIM! perdão, não sou frontender 😅

Reply:
Modificado, obrigado!